src/main/update/launcher-update.ts

```diff
@@ -16,11 +15,6 @@ export async function launcherUpdate(
   update: IUpdate,
   listeners: IUpdateOptions
 ) {
-  if (!getConfig("UseUpdate", process.env.NODE_ENV === "production")) {
```

Comment:
This got deleted, but it doesn't seem to be added anywhere else in this PR. Did we move this logic to other places before this?

Reply:
yes, top of the `main.ts`(127 line)

Reply:
Ooh I see. Thank you for checking in!